src/tools/compiletest/src/runtest.rs

```diff
@@ -3773,7 +3773,12 @@ impl<'test> TestCx<'test> {
 
         let mut stage_std_path = PathBuf::new();
         stage_std_path.push(&build_root);
-        stage_std_path.push(format!("{}-std", stage));
+        if stage == "stage0" {
+            stage_std_path.push("stage0-std");
+        } else {
+            // stage2 reuses stage1-std.
+            stage_std_path.push("stage1-std");
```

Comment:
With `full-bootstrap=true` I believe it doesn't get reused.

Reply:
In that case, do I check for full-bootstrap and then point to stage2-std?

Reply:
By the way why are you looking at `stage*-std/<target>/releasw/deps` instead of `stage*/lib` or `stage*/lib/rustlib/<targey>/lib`?

Reply:
> By the way why are you looking at `stage*-std/<target>/releasw/deps` instead of `stage*/lib` or `stage*/lib/rustlib/<targey>/lib`?

(When I was originally implementing it, I just threw whatever worked together lol, I thought I needed to look at `stage*-std/<target>/releasw/deps`)

Reply:
`stage*-std` contains just a standard library build or is non-existent when copying from an earlier stage, while `stage*` has the full sysroot including both rustc and whichever standard library this rustc will compile code against.

Reply:
`stage*/lib` seemed to have worked